### PR TITLE
Fix pyprocessing attribute error

### PR DESCRIPTION
**Pull request by PeanutbutterWarrior**
Fix a bug where getting any item from PyProcessing.namespace return None. Added default values, and a temporary icon.

### CODE REVIEW

File: pyprocessing/renderer/tkinter/renderer.py [hunk 1]
```diff
@@ -15,14 +15,14 @@ def init(self):
...
         x, y = self.pp.namespace.window_offset
         geometry = f"{w + 2}x{h + 20}+{x}+{y}"
-        self.pyprocessing.logger.info(
+        self.pp.logger.info(
             'Initializing window with geometry %s', geometry
         )
         self.root.geometry(geometry)
         self.root.title(self.pp.namespace.window_title)
-        self.root.resizable(self.pp.namespace.window_resizable)
+        self.root.resizable(self.pp.namespace.window_resizable, self.pp.namespace.window_resizable)
```
Comment (sbordeyne) [inline pyprocessing/renderer/tkinter/renderer.py]: better keep it like it was here, but unpack the tuple instead.

File: pyprocessing/pyprocessing.py [hunk 2]
```diff
@@ -20,14 +21,18 @@ class PPNamespace(dict):
         'fill': '#ffffff',
         'width': 640,
         'height': 480,
+        'window_offset': (2, 2),
+        'window_title': 'pyprocessing',
+        'window_resizable': False,
+        'window_icon': PImage('pyprocessing/images/icon16x16.png')
```
Comment (sbordeyne) [inline pyprocessing/pyprocessing.py]: have you tested that the path does not fail when installing (python setup.py install) and importing the package from a different location ?

**Review by sbordeyne — CHANGES_REQUESTED** [review]
caught a couple of things while reviewing,

File: pyprocessing/pyprocessing.py (hunk 2 above)
Comment (PeanutbutterWarrior) [reply pyprocessing/pyprocessing.py]: It works now, but I'm not sure if my solution is the best way to do it

Comment (sbordeyne) [reply pyprocessing/pyprocessing.py]: Ideally, PImage should be able to take in an image in base64 format, thus eliminating the problem altogether.

I'll see if I can make a commit that adds that feature, and fix that line of code then.

**Pull request merged**